roles/ec2_provision/tasks/provision_instance.yml

```diff
@@ -71,6 +71,16 @@
     - set_fact:
         suggested_ami: "{{ ami.images | map(attribute='image_id') | list | first }}"
 
+    - set_fact:
+        inst_tags:
+          creator_arn: "{{ caller_facts.arn }}"
+          user: "{{ ansible_user }}"
+          Name: "{{ ansible_user }}-{{ instance_name }}"
+
+    - set_fact:
+        inst_tags: "{{ inst_tags | combine(additional_tags) }}"
+      when: additional_tags
```

Comment:
Will the conditional evaluate correctly if additional_tags is not defined?

Reply:
Fixed by 65a0231